tutor/templates/local/docker-compose.yml

```diff
@@ -171,6 +171,8 @@ services:
       {%- endfor %}
     depends_on:
       - lms
+    profiles:
```

Comment:
I wasn't familiar with profiles, to be honest. I like the idea of not running workers in dev, but I'd like to avoid making a breaking change to the filter API, and also changes across many python functions.

Instead, could we simply move the lms-worker and cms-worker declarations from docker-compose.yml to docker-compose.prod.yml?

Reply:
I don't think it's going to be that simple.  There are other places in the code that assume those service definitions exist.  For instance, https://github.com/overhangio/tutor/blob/master/tutor/commands/compose.py#L289.  Plus, I suspect profiles might be useful down the road, woudn't you say so?

If you're dead set against it, though, there is another way: we can just set the worker services in the dev compose override file to a profile like "donotstart".  We get the same thing this PR provides, but without actually supporting different profiles.